claims.go

```diff
@@ -37,7 +37,7 @@ func (c StandardClaims) Valid() error {
 	// default value in Go, let's not fail the verification for them.
 	if c.VerifyExpiresAt(now, false) == false {
 		delta := time.Unix(now, 0).Sub(time.Unix(c.ExpiresAt, 0))
-		vErr.Inner = fmt.Errorf("token is expired by %v", delta)
+		vErr.Inner = fmt.Errorf("Token is expired by %v", delta)
```

Comment:
A colleague pointed out error messages shouldn't be capitalized: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

So the right thing would be to change StandardClaims to lowercase instead of this to capitalized.